configurations/obkv-table-default.toml

```diff
@@ -50,3 +51,11 @@ LogLevel = "info"
 EnableRerouting = false
 MaxConnectionAge = 0
 EnableSLBLoadBalance = false
+
+[LogConfig]
+LogFile = "${Absolute_path}/obkv-table-client-go/logs/"
```

Comment:
The default log directory should be set to the current directory, not our project directory